ts/components/screens/BadgeComponent.tsx

```diff
@@ -0,0 +1,22 @@
+/**
+ * A component to render the circolar badge
+ * TODO: use the same component on all lists (messages, services, transaction)
```

Comment:
Every TODO must be associated with a pivotal story.

Reply:
Done https://github.com/teamdigitale/io-app/pull/1175/commits/fcb08677d4f260ad9913007dc470940fd9db9d87